<?xml version="1.0"?>
<launch>

    <node pkg="rbe500_project" type="rrp_ik_server.py" name="rrp_ik_server" output="screen"/>

    <node pkg="rbe500_project" type="rrp_pid.py" name="rrp_pid" output="screen"/>

</launch>
